src/js/components/RangeSelector/RangeSelector.js

```diff
@@ -63,8 +63,10 @@ const RangeSelector = forwardRef(
 
     const change = useCallback(
       (nextValues) => {
-        setValues(nextValues);
-        if (onChange) onChange(nextValues);
+        const newRange = nextValues.map((n) => Number(Number(n).toFixed(2)));
```

Comment:
Thanks for looking at this issue! Could there be issues with selecting `2` as the fixed length/are we being opinionated? What if the step is 0.135? etc.

Reply:
Do you think that it would be good to capture the number of decimal places from step number informed?